src/Couchbase.Lite.Shared/API/Database/Database.cs

```diff
@@ -1365,6 +1397,29 @@ private void VerifyDB([NotNull]Document document)
             }
         }
 
+        private void PurgeDocById(string id)
+        {
+            ThreadSafety.DoLockedBridge(err =>
+            {
+                return Native.c4db_purgeDoc(_c4db, id, err);
+            });
+        }
+
+        private void PurgeExpired(object state)
+        {
+            var cnt = 0L;
+            ThreadSafety.DoLockedBridge(err =>
+            {
+                CheckOpen();
+                cnt = Native.c4db_purgeExpiredDocs(_c4db, err);
+                Log.To.Database.I(Tag, "{0} purged {1} expired documents", this, cnt);
+                SchedulePurgeExpired(TimeSpan.FromSeconds(1));
+                if (err->code>0 && err->domain>0)
+                    return false;
```

Comment:
Where is this return value going and why is it `false` for success?

Reply:
it's false when error code and domain are greater than 0

Reply:
Oh, I saw it as `=` somehow, but regardless this return value is unused.  Just remove the logic for returning true or false and call with `LiteCoreBridge.Check` instead